applications_tests/gls_sharp_navier_stokes_3d/cavatappi_composite_test.output

```diff
@@ -17,5 +17,5 @@ Steady iteration :        1/1
 +------------------------------------------+
 |  Force  summary particles                |
 +------------------------------------------+
-particle_ID  time    T_x     T_y    T_z     f_x    f_y    f_z    f_xv    f_yv   f_zv   f_xp    f_yp   f_zp  
-          0 1.0000 -0.6968 -4.4547 2.5233 49.1749 1.0278 7.1728 26.3108 0.3030 0.1214 22.8641 0.7248 7.0514
\ No newline at end of file
+particle_ID  time   T_x     T_y     T_z      f_x    f_y    f_z    f_xv    f_yv    f_zv    f_xp   f_yp    f_zp
+          0 1.0000 0.8377 -1.0770 -12.4452 36.4788 9.4690 8.0446 27.5955 -2.0269 -0.8735 8.8833 11.4959 8.9182
```

Comment:
Wow the results for this one has indeed changed dramatically.... Are you sure this is a good test?

Reply:
I'm not sure I have to admit. It's nice to have I'd say but not robust enough. I added an lines to the unit test for composites shapes.